strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout/strongbox-storage-maven-layout-provider/src/test/java/org/carlspring/strongbox/providers/repository/MavenProxyRepositoryProviderTestIT.java

```diff
@@ -168,15 +152,35 @@ public void shouldBeAbleToProvideFilesFromOracleMavenRepoWithHttpsAndAuthenticat
     @Test
-    public void whenDownloadingArtifactMetadaFileShouldAlsoBeResolved(TestInfo testInfo)
+    public void whenDownloadingArtifactMetadaFileShouldAlsoBeResolved(@MavenRepository(storageId = STORAGE_ID,
+                                                                                       repositoryId = REPOSITORY_ID + "-whenDownloadingArtifactMetadaFileShouldAlsoBeResolved")
+                                                                      Repository repository1,
+                                                                      @MavenRepository(storageId = STORAGE_ID,
+                                                                                       repositoryId = CENTRAL_REPOSITORY_ID + "-whenDownloadingArtifactMetadaFileShouldAlsoBeResolved")
+                                                                      Repository repository2,
+                                                                      @TestRepository(layout = LAYOUT_NAME,
+                                                                                      storageId = STORAGE_ID,
+                                                                                      repositoryId = REPOSITORY_ID + "-whenDownloadingArtifactMetadaFileShouldAlsoBeResolved",
+                                                                                      setup = MavenIndexedRepositorySetup.class)
+                                                                      @Remote(url = REMOTE_URL)
+                                                                      Repository proxyRepository1,
+                                                                      @TestRepository(layout = LAYOUT_NAME,
+                                                                                      storageId = STORAGE_ID,
+                                                                                      repositoryId = CENTRAL_REPOSITORY_ID + "-whenDownloadingArtifactMetadaFileShouldAlsoBeResolved",
+                                                                                      setup = MavenIndexedRepositorySetup.class)
+                                                                      @Remote(url = CENTRAL_URL)
+                                                                      Repository proxyRepository2)
             throws Exception
     {
+
         String storageId = STORAGE_ID;
-        String repositoryId = getRepositoryName(REPOSITORY_ID, testInfo);
+        String repositoryId = REPOSITORY_ID + "-whenDownloadingArtifactMetadaFileShouldAlsoBeResolved";
 
         assertStreamNotNull(storageId, repositoryId,
-                            "org/carlspring/properties-injector/1.1/properties-injector-1.1.jar");
+                "org/carlspring/properties-injector/1.1/properties-injector-1.1.jar");
 
         Storage storage = getConfiguration().getStorage(storageId);
         Repository repository = storage.getRepository(repositoryId);
```

Comment:
You don't need to declare those variables. You can use `proxyRepository1` to use it in the next sentence.

Reply:
> @pyashwanthkumar ,
> 
> Could you please use the text of the pull request template and fill in the details instead, please? Thanks! :)

This is done. I have seen the content in another PR and I thought I copy-pasted it in my PR mistakenly. Didn't know it was a template. :)